routes/api.php

```diff
@@ -18,4 +18,4 @@
     return $request->user();
 });
 
-Route::resource('user', UserController::class);
+Route::resource('v1/user', UserController::class);
```

Comment:
please use this approach instead:
Route::prefix('v1')->group(function () {
    Route::resource('/user', UserController::class);
});